code/modules/research/xenoarchaeology/artifact/artifact_necromarker.dm

```diff
@@ -0,0 +1,94 @@
+#define NECROMARKER_MIN_WHISPER_INTERVAL 200
+#define NECROMARKER_MAX_WHISPER_INTERVAL 450
+/obj/machinery/necromarker
+    name = "mysterious sculpture"
+    desc = "A mysterious scultpure of spiral spines, covered in runes."
+    icon = 'icons/xenoarch_icons/necromarker.dmi'
+    icon_state = "black"
+    density = 1
+
+    var/ticks_not_whispered = 0
+    var/next_whisper = 300
+    var/whispers = list("...bring us flesh...", "...make us whole...", "...we must be whole...", "...join us in unity...", "...one mind, one soul, one flesh...", "...MAKE US WHOLE...")
+    var/list/mob/dead/observer/candidates = list()
+
+    machine_flags = WRENCHMOVE
+
+/obj/machinery/necromarker/New()
+    . = ..()
+
+/obj/machinery/necromarker/MouseDrop_T(mob/M as mob, mob/user as mob)
+    if(!istype(M) || isobserver(user))
+        return
+    if(Adjacent(user))
+        Consume(M)
+
+/obj/machinery/necromarker/proc/Consume(mob/M as mob, mob/user as mob)
+    if(anchored && ismob(M) && Adjacent(M) && iscarbon(M) && !istype(M, /mob/living/simple_animal/hostile/monster/necromorph/))
+        var/mob/living/simple_animal/hostile/monster/necromorph/Z = new(src.loc)
+        if(M.ckey)
+            // Z.ckey = M.ckey
+            if(M.mind)
+                M.mind.transfer_to(Z)
+        else
+            for(var/mob/dead/observer/O in candidates)
+                if(O && O.mind && O.ckey)
+                    O.mind.transfer_to(Z)
+                    Z.ckey = O.ckey // Because ghosts don't get key changes
+                    candidates -= O
+                    break
+                else
+                    candidates -= O
+        visible_message("<span class='warning'>[src] spins the flesh and bone of [M] into a hellish monstrosity!</span>")
```

Comment:
Exxion, no, it's not.

If [src] is capitalized then it won't add a the, it only adds the automatically to \improper named objects (which is true if the word isn't capitalized).